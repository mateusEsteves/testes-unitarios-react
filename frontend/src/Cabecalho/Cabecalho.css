.cabecalho {
  width: 100%;
  height: 72px;
  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
  grid-area: header;
  position: fixed;
  background-color: #ad75e6;
  z-index: 1;
  display: flex;
}

.cabecalho .cabecalho__titulo {
  margin: auto 0 auto 12px;
  color: white;
  font-size: 1.7rem;
}
