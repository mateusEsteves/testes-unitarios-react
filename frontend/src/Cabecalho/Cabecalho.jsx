import './Cabecalho.css';

export default function Cabecalho() {
  return (
    <header className="cabecalho">
      <h1 className="cabecalho__titulo">Receitas legais para fazer em casa</h1>
    </header>
  );
}
