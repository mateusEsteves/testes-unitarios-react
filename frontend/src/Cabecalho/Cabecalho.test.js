import { render } from '@testing-library/react';
import Cabecalho from './Cabecalho';

it('Deve renderizar corretamente', () => {
  const { baseElement } = render(<Cabecalho />);
  expect(baseElement).toMatchSnapshot();
});
