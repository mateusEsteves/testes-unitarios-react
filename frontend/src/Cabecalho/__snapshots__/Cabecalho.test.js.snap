// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deve renderizar corretamente 1`] = `
<body>
  <div>
    <header
      class="cabecalho"
    >
      <h1
        class="cabecalho__titulo"
      >
        Receitas legais para fazer em casa
      </h1>
    </header>
  </div>
</body>
`;
